core/src/test/java/com/sequenceiq/cloudbreak/service/image/ImageCatalogServiceTest.java

```diff
@@ -95,6 +97,8 @@
 
     private static final String V2_CATALOG_FILE = "com/sequenceiq/cloudbreak/service/image/cb-image-catalog-v2.json";
 
+    private static final String V3_CATALOG_FILE = "com/sequenceiq/cloudbreak/service/image/cb-image-catalog-v3.json";
```

Comment:
Please test the solution with a freeipa image catalog as well. It is needed as freeipa contains less parameters for a particular image so we should make sure we won't fail on it eg by a NPE.

Reply:
New freeipa catalog is provided for the unit tests.